tools/test/examples/cloud_examples_cmake.json

```diff
@@ -15,6 +15,38 @@
       "baud_rate": 115200,
       "compare_log": ["mbed-os-example-for-aws/tests/aws.log"],
       "auto-update" : true
+    },
```

Comment:
Could we delete `cloud_examples_cmake.json` in this PR? CI can use `cloud_examples.json` without running into failures, even for CMake.

Reply:
I thought about that. It's easier to update the cloud_examples_cmake.json for now as it involves no change in CI scripts. We can delete the examples_cmake.json and cloud_examples_cmake.json together?

Reply:
yes, we should. When can we do it?

Reply:
When we have all the examples passing in cmake. Now just one left, mbed-os-example-mesh-minimal